src/Apitizer/Types/Filter.php

```diff
@@ -153,7 +137,7 @@ public function byAssociation(string $relation, string $key = null): self
      */
     public function search($fields): self
     {
-        $this->expect('string');
+        $this->expect();
```

Comment:
This doesn't do anything anymore, as it just returns the Picker object. Instead, it should probably set the type to string and the expectArray to false.

Reply:
`expect()` already sets the expectArray to false, and instead of setting the type to string there the same result can be achieved by doing `$this->expect()->string();`

Reply:
Alright, then let's change it to `$this->expect()->string()`